### PR TITLE
ran aspell

### PR DESCRIPTION
**Pull request by JensTimmerman**

### CODE REVIEW

**Comment by boegel** [comment]
unmergable, needs to be fixed first

**Comment by JensTimmerman** [comment]
mergable now

File: perfexpert/ch07_optimization_patterns.tex [hunk 1]
```diff
@@ -1,7 +1,7 @@
-\chapter{Annex 2: Optimization Patterns}
+\chapter{Annex 2: Optimisation Patterns}
 \label{ch:ch07_optimization_patterns}
 
-\section{Memory Access optimizations}
+\section{Memory Access optimisation's}
```
Comment (boegel) [inline perfexpert/ch07_optimization_patterns.tex]: no way this is correct

**Comment by boegel** [comment]
`optimisations` shouldn't be replaced by `optimisation's`

**Comment by boegel** [comment]
looks good otherwise

**Comment by JensTimmerman** [comment]
weird, aspell had that one wrong... fixed now.

**Comment by boegel** [comment]
fine to merge by me then

**Pull request merged**